arp.rflx

```diff
@@ -0,0 +1,39 @@
+with Ethernet;
+with IPv4;
+
+package ARP is
+
+   -- RFC 826
+
+   type Hardware_Address_Space is
+   (
+      Hrd_Ethernet =>   1,
+      Hrd_HW_Exp1  =>  36,
+      Hrd_HW_Exp2  => 256
+   ) with Size => 16;
```

Comment:
We should decide on a consistent coding style. There are several variants for formatting enumerations just in this PR. None of the variants used in this PR match the one I have used in other specifications:

```Ada
   type Handshake_Type is (
      HANDSHAKE_CLIENT_HELLO => 1,
      HANDSHAKE_SERVER_HELLO => 2,
      HANDSHAKE_NEW_SESSION_TICKET => 4,
      HANDSHAKE_END_OF_EARLY_DATA => 5,
      HANDSHAKE_ENCRYPTED_EXTENSIONS => 8,
      HANDSHAKE_CERTIFICATE => 11,
      HANDSHAKE_CERTIFICATE_REQUEST => 13,
      HANDSHAKE_CERTIFICATE_VERIFY => 15,
      HANDSHAKE_FINISHED => 20,
      HANDSHAKE_KEY_UPDATE => 24,
      HANDSHAKE_COMPRESSED_CERTIFICATE => 25,  -- draft-ietf-tls-certificate-compression
      HANDSHAKE_MESSAGE_HASH => 254
   ) with Size => 8;
```

Reply:
See https://github.com/Componolit/ada-style/issues/24.

Reply:
Fixed in ace51fd.